src/open_inwoner/openzaak/statuses.py

```diff
@@ -33,7 +34,24 @@ def fetch_status_history(case_url: str) -> List[Status]:
     return statuses
 
 
-def fetch_status_types(zaaktype: str) -> List[StatusType]:
+def fetch_specific_statuses(status_urls: List[str]) -> List[Status]:
+    client = build_client("zaak")
+
+    if client is None:
+        return []
+
+    with parallel() as executor:
+        _statuses = executor.map(
+            lambda url: client.retrieve("status", url=url),
+            status_urls,
+        )
+
+    statuses = factory(Status, list(_statuses))
+
+    return statuses
+
+
+def fetch_status_types(*case_type: str) -> List[StatusType]:
```

Comment:
Why do you user asterisk here?

Reply:
Because I am using this function with and without a variable (accounts/views/cases.py). Do you mean I should use a keyword argument?

Reply:
yes, please use a keyword argument with the default value. When * is used The tuple is initialized (https://docs.python.org/3/reference/compound_stmts.html#function-definitions), I don't think we need it here.